saleor/product/models.py

```diff
@@ -407,6 +421,12 @@ def annotate_quantities(self):
             ),
         )
 
+    def prefetched_for_webhook(self):
+        return self.prefetch_related(
+            "attributes__values",
+            "attributes__assignment__attribute",
```

Comment:
Just to confirm: What about `media`? I see that payload generated for variant uses it.

Reply:
Yes, I tested it.

Reply:
We have N queries here, where N is the number of media assigned to variant. 
First case: Variant without prefetch and some media objects. For each media obj, it generates one query.
Second: Prefetch with const number of queries
![image](https://user-images.githubusercontent.com/23282005/113691863-32d1bb80-96cd-11eb-9230-138454385c0c.png)

Reply:
Fixed.